lib/project/util.coffee

```diff
@@ -25,3 +26,20 @@ module.exports =
       "#{_.pluralize(matchCount, 'result')} found in #{_.pluralize(pathCount, 'file')} for <span class=\"highlight-info\">#{@sanitizePattern(findPattern)}</span>"
     else
       "No #{if replacedPathCount? then 'more' else ''} results found for '#{@sanitizePattern(findPattern)}'"
+
+  parseSearchResult: ->
+    searchResult = []
+    summary = $('span.preview-count', 'div.preview-pane').text()
+    searchResult.push summary, ''
+
+    $('ol.results-view.list-tree>li.path').each ->
+      path = $('span.path-name', this).text()
+      matches = $('span.path-match-number', this).text()
+      searchResult.push path + ' ' + matches
+
+      $('li.search-result', this).filter(':visible').each ->
+        lineNumber = $('span.line-number', this).text()
+        preview = $('span.preview', this).text()
+        searchResult.push '\t' + lineNumber + '\t' + preview
+      searchResult.push ''
+    searchResult.join('\n')
```

Comment:
Above part could be rewritten with plain DOM API to remove the `space-pen` dependency with minimal effort. Might help with getting this merged.